pages/release-process.md

```diff
@@ -1,111 +1,113 @@
-
-    ```shell
-    # Creates 1.0.0-beta-1-RC1 from 1.0.0-beta-1 
-    git tag 1.0.0-beta-1-RC1 1.0.0-beta-1
-    # Delete 1.0.0-beta-1
-    git tag -d 1.0.0-beta-1
-    # Push RC tag to fork
-    git push origin 1.0.0-beta-1-RC1
-    ```
-
- 7. The artifacts will be staged in nexus OSS. Send a message to the devs to let them know a release is staged. 
-
- 8. Give enough time (Apache recommends 72 hours) for everyone to check out the distribution tarball from Sonatype OSS and verify signatures, hashes, functional tests, etc... Sonatype OSS does have it's own validation of the artifacts which includes verifying a valid GPG signature, though I do not believe it verifies that it belonged to a trusted committer so that'll need to be done by other committers.
-
-When consensus has been reached on a release candidate, follow the steps below to complete the release using the chosen RC:
-
- 1. Merge your RC branch into master and push those commits upstream.  Afterwards, you can delete your RC branch.
-
-    ```shell
-    git checkout master
-    git merge 1.0.0-beta-RC
-    git push upstream master
-    git branch -d 1.0-0-beta-RC
-    ```
-
- 2. [Release the artifacts] in Sonatype OSS so that they get published in Maven Central.  You can drop any staging repos for RCs that were not chosen. 
-
- 3. Create a signed tag for the release from the chosen RC tag and push to upstream repo:
-
-    ```shell
-    # Create signed tag from RC2 tag.
-    # You may need to use -u <key-id> to specify GPG key
-    git tag -s 1.0.0-beta-1 1.0.0-beta-1-RC2
-    # Push signed tag to upstream repo
-    git push upstream 1.0.0-beta-1
-    ```
-
- 4. Attach Fluo tarball to GitHub release page.
-
- 5. Remove all RC tags
-
-    ```shell
-    # Remove tag locally
-    git tag -d 1.0.0-beta-1-RC1
-    # Remove tag on fork
-    git push --delete origin 1.0.0-beta-1-RC1
-    ```
- 6.  View the [website README] for instructions on how to generate Javadocs and documentation using the released tag.  Submit PR to the website repo to publish.
- 
- 7.  Send an email to `dev@fluo.incubator.apache.org` announcing new release.
-
-[website README]: https://github.com/apache/incubator-fluo-website/blob/master/README.md
-[documentation]: http://central.sonatype.org/pages/apache-maven.html
-[sonatype account]: https://issues.sonatype.org/
-[Release the artifacts]: http://central.sonatype.org/pages/releasing-the-deployment.html
+---
+layout: page
+title: Release Process
+permalink: /release-process/
+---
+
+### Initial Setup
+
+Before you can release Fluo or Fluo Recipes, you will need a GPG key. For information on generating
+a key look at this [ASF GPG page](https://www.apache.org/dev/openpgp.html).  After generating a key,
+add it to the [KEYS](https://www.apache.org/dist/incubator/fluo/KEYS) file.
```

Comment:
Could mention that instructions are provided at top of KEYS file on how to do this.